frontend/src/components/admin/question-creation/QuestionElementItem.tsx

```diff
@@ -152,10 +151,12 @@ const QuestionElementItem = ({
         </Box>
         {renderQuestionContent(content)}
         <IconButton
+          _hover={{ color: "blue.100" }}
+          aria-label="delete-icon-button"
```

Comment:
Does this read out fine to screen readers with all the hyphens? Normally I'd put spaces instead to make it more readable

Reply:
see latest commit